src/config.rs

```diff
@@ -150,11 +151,12 @@ impl Config {
     fn check_for_timezone(self: Config) -> Result<Config, String> {
         if self.timezone.is_none() {
             let desc = "Please select your timezone";
-            let options = TZ_VARIANTS
+            let mut options = TZ_VARIANTS
                 .to_vec()
                 .iter()
                 .map(|v| v.to_string())
                 .collect::<Vec<String>>();
+            options.sort();
```

Comment:
I think that the fundamental issue with the test weirdness is that the TZ_VARIANTS were not consistently sorted.